### DISCUSSION TITLE
Cannot get UI to load (after switching model)

### DISCUSSION THREAD

**Discussion by raffetazarius** [vladmandic/sdnext#3126] [Q&A]
I had a fresh installation of SD.Next running fine on my MacBook Pro M3 Max 128GB 8TB.  

The only thing I did was switched the model, and it seems to crash the UI and I cannot get it load again.

I've tried loading using both the Original and Diffusers backends (using the --backend arg), but no luck.

Is there any way I can get this going again without doing another reinstall?

-----

privateuser@Users-MBP automatic % ./webui.sh  --disable-queue
Create and activate python venv
Launching launch.py...
19:20:44-133645 INFO     Starting SD.Next                                                                                                                     
19:20:44-135242 INFO     Logger: file="/Users/privateuser/automatic/sdnext.log" level=INFO size=19863 mode=append                                           
19:20:44-135765 INFO     Python 3.10.13 on Darwin                                                                                                             
19:20:44-154780 INFO     Version: app=sd.next updated=2024-05-07 hash=e081f232 branch=master url=https://github.com/vladmandic/automatic/tree/master          
19:20:44-570198 INFO     Platform: arch=arm64 cpu=arm system=Darwin release=23.4.0 python=3.10.13                                                             
19:20:44-597896 WARNING  Modified files: ['models/Reference/DeepFloyd--IF-I-M-v1.0.jpg', 'models/Reference/PixArt-alpha--PixArt-XL-2-1024-MS.jpg',            
                         'models/Reference/PixArt-alpha--PixArt-XL-2-512x512.jpg', 'models/Reference/SimianLuo--LCM_Dreamshaper_v7.jpg',                      
                         'models/Reference/TempestV0.1-Artistic.jpg', 'models/Reference/XCLiu--instaflow_0_9B_from_sd_1_5.jpg',                               
                         'models/Reference/amused--amused-256.jpg', 'models/Reference/amused--amused-512.jpg',                                                
                         'models/Reference/damo-vilab--text-to-video-ms-1.7b.jpg', 'models/Reference/dreamshaperXL_v21TurboDPMSDE.jpg',                       
                         'models/Reference/dreamshaper_8.jpg', 'models/Reference/etri-vilab--koala-700m-llava-cap.jpg',                                       
                         'models/Reference/juggernautXL_v9Rdphoto2Lightning.jpg', 'models/Reference/juggernautXL_v9Rundiffusionphoto2.jpg',                   
                         'models/Reference/juggernaut_reborn.jpg', 'models/Reference/kandinsky-community--kandinsky-2-1.jpg',                                 
                         'models/Reference/kandinsky-community--kandinsky-2-2-decoder.jpg', 'models/Reference/kandinsky-community--kandinsky-3.jpg',          
                         'models/Reference/latent-consistency--lcm-sdxl.jpg', 'models/Reference/playgroundai--playground-v1.jpg',                             
                         'models/Reference/playgroundai--playground-v2-1024px-aesthetic.jpg', 'models/Reference/playgroundai--playground-v2-256px-base.jpg',  
                         'models/Reference/playgroundai--playground-v2-512px-base.jpg', 'models/Reference/playgroundai--playground-v2.5-1024px-aesthetic.jpg',
                         'models/Reference/salesforce--blipdiffusion.jpg', 'models/Reference/sd_xl_base_1.0.jpg', 'models/Reference/sdxl_turbo.jpg',          
                         'models/Reference/segmind--SSD-1B.jpg', 'models/Reference/segmind--SegMoE-2x1-v0.jpg', 'models/Reference/segmind--SegMoE-4x2-v0.jpg',
                         'models/Reference/segmind--SegMoE-SD-4x2-v0.jpg', 'models/Reference/segmind--Segmind-Vega.jpg',                                      
                         'models/Reference/segmind--tiny-sd.jpg', 'models/Reference/stabilityai--stable-cascade.jpg',                                         
                         'models/Reference/stabilityai--stable-diffusion-2-1-base.jpg', 'models/Reference/stabilityai--stable-diffusion-2-1.jpg',             
                         'models/Reference/stabilityai--stable-video-diffusion-img2vid-xt.jpg',                                                               
                         'models/Reference/stabilityai--stable-video-diffusion-img2vid.jpg', 'models/Reference/thu-ml--unidiffuser-v1.jpg',                   
                         'models/Reference/v1-5-pruned-fp16-emaonly.jpg', 'models/Reference/warp-ai--wuerstchen.jpg', 'models/VAE-approx/model.pt',           
                         'models/karlo/ViT-L-14_stats.th']                                                                                                    
19:20:44-610200 INFO     Extensions: disabled=[]                                                                                                              
19:20:44-611247 INFO     Extensions: enabled=['sd-extension-chainner', 'sd-webui-controlnet', 'stable-diffusion-webui-rembg', 'sd-webui-agent-scheduler',     
                         'Lora', 'sd-extension-system-info', 'stable-diffusion-webui-images-browser'] extensions-builtin                                      
19:20:44-612694 INFO     Extensions: enabled=[] extensions                                                                                                    
19:20:44-613287 INFO     Startup: quick launch                                                                                                                
19:20:44-613736 INFO     Verifying requirements                                                                                                               
19:20:44-622136 INFO     Verifying packages                                                                                                                   
19:20:44-623200 INFO     Extensions: disabled=[]                                                                                                              
19:20:44-623680 INFO     Extensions: enabled=['sd-extension-chainner', 'sd-webui-controlnet', 'stable-diffusion-webui-rembg', 'sd-webui-agent-scheduler',     
                         'Lora', 'sd-extension-system-info', 'stable-diffusion-webui-images-browser'] extensions-builtin                                      
19:20:44-624717 INFO     Extensions: enabled=[] extensions                                                                                                    
19:20:44-627119 INFO     Command line args: ['--disable-queue'] disable_queue=True                                                                            
19:20:46-642281 INFO     Load packages: {'torch': '2.3.0', 'diffusers': '0.27.0', 'gradio': '3.43.2'}                                                         
19:20:47-113992 INFO     Engine: backend=Backend.DIFFUSERS compute=mps device=mps attention="Scaled-Dot-Product" mode=no_grad                                 
19:20:47-116001 INFO     Device:                                                                                                                              
19:20:47-410838 INFO     Available VAEs: path="models/VAE" items=2                                                                                            
19:20:47-411494 INFO     Disabled extensions: ['sd-webui-controlnet']                                                                                         
19:20:47-414419 INFO     Available models: path="models/Stable-diffusion" items=55 time=0.00                                                                  
19:20:47-453286 INFO     LoRA networks: available=23 folders=2                                                                                                
19:20:47-681901 INFO     Extension: script='extensions-builtin/sd-webui-agent-scheduler/scripts/task_scheduler.py' Using sqlite file:                         
                         extensions-builtin/sd-webui-agent-scheduler/task_scheduler.sqlite3                                                                   
19:20:47-795134 INFO     UI theme: name="black-teal" style=Auto base=sdnext.css                                                                               
19:20:48-445895 INFO     Extension list is empty: refresh required                                                                                            
19:20:48-686400 INFO     Server queues disabled                                                                                                               
19:20:48-734809 INFO     Local URL: http://127.0.0.1:7860/                                                                                                    
19:20:48-830533 INFO     [AgentScheduler] Total pending tasks: 1                                                                                              
19:20:48-834660 INFO     [AgentScheduler] Executing task task(xtr3jxmms1hezac)                                                                                
19:20:48-835197 INFO     [AgentScheduler] Registering APIs                                                                                                    
19:20:49-013094 INFO     Select: model="SD 1.5/absolutereality_v181 [463d6a9fe8]"                                                                             
19:20:49-037990 INFO     Setting Torch parameters: device=mps dtype=torch.float32 vae=torch.float32 unet=torch.float32 context=no_grad fp16=None bf16=None    
                         optimization=Scaled-Dot-Product                                                                                                      
19:20:49-045087 INFO     Autodetect: model="Stable Diffusion" class=StableDiffusionPipeline file="/Users/privateuser/automatic/models/Stable-diffusion/SD   
                         1.5/absolutereality_v181.safetensors" size=2034MB                                                                                    
19:20:49-084150 INFO     Startup time: 4.41 torch=1.64 gradio=0.34 libraries=0.76 extensions=0.36 ui-en=0.08 ui-control=0.05 ui-settings=0.11                 
                         ui-extensions=0.48 ui-defaults=0.06 app-started=0.26                                                                                 
19:20:51-033540 INFO     Load embeddings: loaded=14 skipped=11 time=0.16                                                                                      
19:20:51-158657 INFO     Load model: time=2.01 load=2.00 native=512 {'ram': {'used': 4.07, 'total': 128.0}}                                                   
19:20:51-161068 INFO     Select: model="SD 1.5/cirrostratus_v25 [9abd72c940]"                                                                                 
19:20:51-672081 INFO     Setting Torch parameters: device=mps dtype=torch.float32 vae=torch.float32 unet=torch.float32 context=no_grad fp16=None bf16=None    
                         optimization=Scaled-Dot-Product                                                                                                      
19:20:51-672862 INFO     Autodetect: model="Stable Diffusion" class=StableDiffusionPipeline file="/Users/privateuser/automatic/models/Stable-diffusion/SD   
                         1.5/cirrostratus_v25.safetensors" size=2193MB                                                                                        
19:20:53-089710 INFO     Load embeddings: loaded=14 skipped=11 time=0.14                                                                                      
19:20:53-212810 INFO     Load model: time=1.42 load=1.42 native=512 {'ram': {'used': 4.31, 'total': 128.0}}                                                   
19:20:53-224745 INFO     Base: class=StableDiffusionImg2ImgPipeline                                                                                           
Progress ?it/s                                              0% 0/20 00:00 ? Base/AppleInternal/Library/BuildRoots/ce725a5f-c761-11ee-a4ec-b6ef2fd8d87b/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShaders/MPSCore/Types/MPSNDArray.mm:788: failed assertion `[MPSNDArray initWithDevice:descriptor:] Error: total bytes of NDArray > 2**32'
zsh: abort      ./webui.sh --disable-queue
privateuser@Users-MBP automatic % /Users/privateuser/.pyenv/versions/3.10.13/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '

**Comment by vladmandic** [comment]
model `cirrostratus_v25` you've tried has a really strange size for sd15 model and seems like apple mps has a limit to max size and it crashes.
but when you restart sdnext, it tries to load same model again.
i suggest to either edit `config json` to remove that model so its no longer auto-loaded on startup and pick some normal model.

**Comment by raffetazarius** [comment]
Thanks Vlad!

I've removed reference to the model from config.json, but the same crash occurs trying to start up SD.Next:

07:25:11-752099 INFO     Base: class=StableDiffusionImg2ImgPipeline                                                                                      
Progress ?it/s                                              0% 0/20 00:00 ? Base/AppleInternal/Library/BuildRoots/ce725a5f-c761-11ee-a4ec-b6ef2fd8d87b/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShaders/MPSCore/Types/MPSNDArray.mm:788: failed assertion `[MPSNDArray initWithDevice:descriptor:] Error: total bytes of NDArray > 2**32'
zsh: abort      ./webui.sh
privateusers@Users-MBP automatic % /Users/privateuser/.pyenv/versions/3.10.13/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
privateuser@Users-MBP automatic % 


<img width="823" alt="Screenshot 2024-05-15 at 07 35 22" src="https://github.com/vladmandic/automatic/assets/73661438/6ebd7390-bde6-4401-8062-cabded668721">

Is there a way to force SD.Next *not* to load the model on start up, using a cmd arg?  Then I can get into the UI to disable auto-loading of a model on startup (so that this doesn't occur again and a restart of the UI actually restores functionality as it should)

**Comment by raffetazarius** [comment]
I've also tried inserting a DIFFERENT model into line 37 of that file, it has no effect on the SD.Next startup sequence.

I've also tried deleting the cache.json file from the automatic folder (I believe this contains hashes of models).

Now a different error occurs and I still can't load SD.Next UI.

----

privateuser@Users-MBP automatic % ./webui.sh  
Create and activate python venv
Launching launch.py...
07:46:12-164431 INFO     Starting SD.Next                                       
07:46:12-166010 INFO     Logger:                                                
                         file="/Users/privateuser/automatic/sdnext.log"       
                         level=INFO size=96102 mode=append                      
07:46:12-166595 INFO     Python 3.10.13 on Darwin                               
07:46:12-185682 INFO     Version: app=sd.next updated=2024-05-07 hash=e081f232  
                         branch=master                                          
                         url=https://github.com/vladmandic/automatic/tree/master
07:46:12-549218 INFO     Platform: arch=arm64 cpu=arm system=Darwin             
                         release=23.4.0 python=3.10.13                          
07:46:12-582009 WARNING  Modified files:                                        
                         ['models/Reference/DeepFloyd--IF-I-M-v1.0.jpg',        
                         'models/Reference/PixArt-alpha--PixArt-XL-2-1024-MS.jpg
                         ',                                                     
                         'models/Reference/PixArt-alpha--PixArt-XL-2-512x512.jpg
                         ',                                                     
                         'models/Reference/SimianLuo--LCM_Dreamshaper_v7.jpg',  
                         'models/Reference/TempestV0.1-Artistic.jpg',           
                         'models/Reference/XCLiu--instaflow_0_9B_from_sd_1_5.jpg
                         ', 'models/Reference/amused--amused-256.jpg',          
                         'models/Reference/amused--amused-512.jpg',             
                         'models/Reference/damo-vilab--text-to-video-ms-1.7b.jpg
                         ', 'models/Reference/dreamshaperXL_v21TurboDPMSDE.jpg',
                         'models/Reference/dreamshaper_8.jpg',                  
                         'models/Reference/etri-vilab--koala-700m-llava-cap.jpg'
                         ,                                                      
                         'models/Reference/juggernautXL_v9Rdphoto2Lightning.jpg'
                         ,                                                      
                         'models/Reference/juggernautXL_v9Rundiffusionphoto2.jpg
                         ', 'models/Reference/juggernaut_reborn.jpg',           
                         'models/Reference/kandinsky-community--kandinsky-2-1.jp
                         g',                                                    
                         'models/Reference/kandinsky-community--kandinsky-2-2-de
                         coder.jpg',                                            
                         'models/Reference/kandinsky-community--kandinsky-3.jpg'
                         , 'models/Reference/latent-consistency--lcm-sdxl.jpg', 
                         'models/Reference/playgroundai--playground-v1.jpg',    
                         'models/Reference/playgroundai--playground-v2-1024px-ae
                         sthetic.jpg',                                          
                         'models/Reference/playgroundai--playground-v2-256px-bas
                         e.jpg',                                                
                         'models/Reference/playgroundai--playground-v2-512px-bas
                         e.jpg',                                                
                         'models/Reference/playgroundai--playground-v2.5-1024px-
                         aesthetic.jpg',                                        
                         'models/Reference/salesforce--blipdiffusion.jpg',      
                         'models/Reference/sd_xl_base_1.0.jpg',                 
                         'models/Reference/sdxl_turbo.jpg',                     
                         'models/Reference/segmind--SSD-1B.jpg',                
                         'models/Reference/segmind--SegMoE-2x1-v0.jpg',         
                         'models/Reference/segmind--SegMoE-4x2-v0.jpg',         
                         'models/Reference/segmind--SegMoE-SD-4x2-v0.jpg',      
                         'models/Reference/segmind--Segmind-Vega.jpg',          
                         'models/Reference/segmind--tiny-sd.jpg',               
                         'models/Reference/stabilityai--stable-cascade.jpg',    
                         'models/Reference/stabilityai--stable-diffusion-2-1-bas
                         e.jpg',                                                
                         'models/Reference/stabilityai--stable-diffusion-2-1.jpg
                         ',                                                     
                         'models/Reference/stabilityai--stable-video-diffusion-i
                         mg2vid-xt.jpg',                                        
                         'models/Reference/stabilityai--stable-video-diffusion-i
                         mg2vid.jpg',                                           
                         'models/Reference/thu-ml--unidiffuser-v1.jpg',         
                         'models/Reference/v1-5-pruned-fp16-emaonly.jpg',       
                         'models/Reference/warp-ai--wuerstchen.jpg',            
                         'models/VAE-approx/model.pt',                          
                         'models/karlo/ViT-L-14_stats.th']                      
07:46:12-593903 INFO     Extensions: disabled=[]                                
07:46:12-594855 INFO     Extensions: enabled=['sd-extension-chainner',          
                         'sd-webui-controlnet', 'stable-diffusion-webui-rembg', 
                         'sd-webui-agent-scheduler', 'Lora',                    
                         'sd-extension-system-info',                            
                         'stable-diffusion-webui-images-browser']               
                         extensions-builtin                                     
07:46:12-596227 INFO     Extensions: enabled=[] extensions                      
07:46:12-596816 INFO     Startup: quick launch                                  
07:46:12-597258 INFO     Verifying requirements                                 
07:46:12-605600 INFO     Verifying packages                                     
07:46:12-606630 INFO     Extensions: disabled=[]                                
07:46:12-607080 INFO     Extensions: enabled=['sd-extension-chainner',          
                         'sd-webui-controlnet', 'stable-diffusion-webui-rembg', 
                         'sd-webui-agent-scheduler', 'Lora',                    
                         'sd-extension-system-info',                            
                         'stable-diffusion-webui-images-browser']               
                         extensions-builtin                                     
07:46:12-608019 INFO     Extensions: enabled=[] extensions                      
07:46:12-610256 INFO     Command line args: []                                  
07:46:14-708523 INFO     Load packages: {'torch': '2.3.0', 'diffusers': '0.27.0', 'gradio': '3.43.2'}                                                                                      
07:46:15-244759 INFO     Engine: backend=Backend.DIFFUSERS compute=mps device=mps attention="Scaled-Dot-Product" mode=no_grad                                                              
07:46:15-246964 INFO     Device:                                                                                                                                                           
07:46:15-532723 INFO     Available VAEs: path="models/VAE" items=2                                                                                                                         
07:46:15-533375 INFO     Disabled extensions: ['sd-webui-controlnet']                                                                                                                      
07:46:15-536210 INFO     Available models: path="models/Stable-diffusion" items=55 time=0.00                                                                                               
07:46:15-579827 INFO     LoRA networks: available=23 folders=2                                                                                                                             
07:46:15-812415 INFO     Extension: script='extensions-builtin/sd-webui-agent-scheduler/scripts/task_scheduler.py' Using sqlite file:                                                      
                         extensions-builtin/sd-webui-agent-scheduler/task_scheduler.sqlite3                                                                                                
07:46:15-925766 INFO     UI theme: name="black-teal" style=Auto base=sdnext.css                                                                                                            
07:46:16-562662 INFO     Extension list is empty: refresh required                                                                                                                         
07:46:16-883937 INFO     Local URL: http://127.0.0.1:7860/                                                                                                                                 
07:46:17-104495 INFO     [AgentScheduler] Total pending tasks: 1                                                                                                                           
07:46:17-108491 INFO     [AgentScheduler] Executing task task(xtr3jxmms1hezac)                                                                                                             
07:46:17-109000 INFO     [AgentScheduler] Registering APIs                                                                                                                                 
07:46:17-192291 INFO     Select: model="SD Turbo/sd_turbo"                                                                                                                                 
07:46:17-193643 INFO     Setting Torch parameters: device=mps dtype=torch.float32 vae=torch.float32 unet=torch.float32 context=no_grad fp16=None bf16=None optimization=Scaled-Dot-Product 
07:46:17-194548 INFO     Startup time: 4.58 torch=1.71 gradio=0.36 libraries=0.81 extensions=0.37 ui-en=0.08 ui-control=0.05 ui-settings=0.11 ui-extensions=0.48 ui-defaults=0.06          
                         launch=0.08 app-started=0.26                                                                                                                                      
07:46:17-196464 INFO     Autodetect: model="Stable Diffusion 2" class=StableDiffusionPipeline file="/Users/privateuser/automatic/models/Stable-diffusion/SD Turbo/sd_turbo.safetensors"  
                         size=4973MB                                                                                                                                                       
text_encoder/config.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 633/633 [00:00<00:00, 681kB/s]
tokenizer/tokenizer_config.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 824/824 [00:00<00:00, 3.01MB/s]
tokenizer/vocab.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.06M/1.06M [00:00<00:00, 4.13MB/s]
tokenizer/merges.txt: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 525k/525k [00:00<00:00, 43.2MB/s]
tokenizer/special_tokens_map.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 460/460 [00:00<00:00, 1.46MB/s]
Calculating hash: /Users/privateuser/automatic/models/Stable-diffusion/SD Turbo/sd_turbo.safetensors ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.2/5.2 GB 0:00:00
07:46:22-516089 ERROR    Model move: device=mps Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta
                         to a different device.                                                                                                                                            
07:46:22-540593 INFO     Load embeddings: loaded=0 skipped=25 time=0.02                                                                                                                    
07:46:22-674658 INFO     Load model: time=5.35 load=5.35 native=768 {'ram': {'used': 0.78, 'total': 128.0}}                                                                                
07:46:22-677179 WARNING  Override not found: checkpoint=SD 1.5/cirrostratus_v25 [9abd72c940]                                                                                               
07:46:22-677665 INFO     Select: model="SD Turbo/sd_turbo [3f067a1b94]"                                                                                                                    
07:46:22-691636 ERROR    Model move: device=mps Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta
                         to a different device.                                                                                                                                            
07:46:22-692249 INFO     Base: class=StableDiffusionImg2ImgPipeline                                                                                                                        
07:46:22-765062 ERROR    Prompt parser encode: Tensor on device mps:0 is not on the expected device meta!                                                                                  
07:46:22-774266 ERROR    Model move: device=mps Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta
                         to a different device.                                                                                                                                            
07:46:22-824124 ERROR    Processing: args={'prompt': [''], 'negative_prompt': [''], 'guidance_scale': 12, 'generator': [<torch._C.Generator object at 0x3acd77a70>],                       
                         'callback_on_step_end': <function process_diffusers.<locals>.diffusers_callback at 0x35b9e1900>, 'callback_on_step_end_tensor_inputs': ['latents',                
                         'prompt_embeds', 'negative_prompt_embeds'], 'num_inference_steps': 41, 'eta': 1.0, 'output_type': 'latent', 'image': [<PIL.Image.Image image mode=RGB             
                         size=1024x1536 at 0x38D253520>], 'strength': 0.5} Cannot copy out of meta tensor; no data!                                                                        
07:46:22-825485 ERROR    Processing: NotImplementedError                                                                                                                                   
╭─────────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ───────────────────────────────────────────────────────────────────────────╮
│ /Users/privateuser/automatic/modules/processing_diffusers.py:426 in process_diffusers                                                                                                 │
│                                                                                                                                                                                         │
│   425 │   │   sd_models.move_model(shared.sd_model, devices.device)                                                                                                                     │
│ ❱ 426 │   │   output = shared.sd_model(**base_args) # pylint: disable=not-callable                                                                                                      │
│   427 │   │   if isinstance(output, dict):                                                                                                                                              │
│                                                                                                                                                                                         │
│ /Users/privateuser/automatic/venv/lib/python3.10/site-packages/torch/utils/_contextlib.py:115 in decorate_context                                                                     │
│                                                                                                                                                                                         │
│   114 │   │   with ctx_factory():                                                                                                                                                       │
│ ❱ 115 │   │   │   return func(*args, **kwargs)                                                                                                                                          │
│   116                                                                                                                                                                                   │
│                                                                                                                                                                                         │
│ /Users/privateuser/automatic/venv/lib/python3.10/site-packages/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py:980 in __call__                              │
│                                                                                                                                                                                         │
│    979 │   │   )                                                                                                                                                                        │
│ ❱  980 │   │   prompt_embeds, negative_prompt_embeds = self.encode_prompt(                                                                                                              │
│    981 │   │   │   prompt,                                                                                                                                                              │
│                                                                                                                                                                                         │
│ /Users/privateuser/automatic/venv/lib/python3.10/site-packages/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py:443 in encode_prompt                         │
│                                                                                                                                                                                         │
│    442 │   │                                                                                                                                                                            │
│ ❱  443 │   │   prompt_embeds = prompt_embeds.to(dtype=prompt_embeds_dtype, device=device)                                                                                               │
│    444                                                                                                                                                                                  │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
NotImplementedError: Cannot copy out of meta tensor; no data!
07:46:22-912771 INFO     Processed: images=0 time=0.23 its=0.00 memory={'ram': {'used': 0.82, 'total': 128.0}}                                                                             
07:46:22-919794 INFO     Select: model="SD Turbo/sd_turbo [3f067a1b94]"                                                                                                                    
07:46:22-930149 INFO     [AgentScheduler] Task queue is empty

**Comment by vladmandic** [comment]
try `webui --ckpt none`
and if that doesn't help, you can manually add `"sd_checkpoint_autoload": false,` to config.json
